config/locales/en.yml

```diff
@@ -1021,7 +1021,7 @@ en:
     condition_goto_page_check_your_answers: "“Check your answers before submitting”."
     condition_goto_page_text: "%{goto_page_position}, “%{goto_page_text}”"
     condition_goto_page_text_with_errors: "[Question not selected]"
-    condition_name: Route %{route_index} of question %{page_index}’s routes
+    condition_name: Question %{page_index}’s route
```

Comment:
In the design it says `Question 2's routes` rather than `Question 2's route`. 

Should it be 'routes' instead of 'route'?

Reply:
Oops, yes it does, good spot, thanks!